CMakeLists.txt

```diff
@@ -1,5 +1,10 @@
 cmake_minimum_required(VERSION 3.11)
-project (uvwasi LANGUAGES C)
+project (
+    uvwasi
+    DESCRIPTION "WASI syscall API built atop libuv"
+    VERSION 0.0.18
```

Comment:
We need a way to bump this when we make a release.

Reply:
I use release-please from google but I don't know if the node org restricts actions in any way. I think we should do this in a future update and probably discuss the release process on the team meeting call.

Reply:
It looks like that is already used in [at least one other repo in the nodejs org](https://github.com/nodejs/node-core-test/blob/main/.github/workflows/release-please.yml), so I say go for it.

Reply:
I've added manually updating this to the release docs but I'll open an issue to automate it.

Reply:
#212